images/dockerregistry/Dockerfile

```diff
@@ -9,6 +9,9 @@ FROM openshift/origin-base
 RUN yum install -y tree findutils epel-release && \
     yum clean all
 
+# The registry doesn't require a privileged user.
+USER 1001
```

Comment:
Although that does increase the risk.  We can live with registry requiring higher scc until 1.1.1